plugin/PlayAssetDelivery/src/androidTest/java/com/google/play/core/godot/assetpacks/ExampleInstrumentedTest.java

```diff
@@ -1,42 +1,40 @@
 /*
```

Comment:
The three Java class files are changed in this commit since the files on the master branch are not yet formatted.